.github/PULL_REQUEST_TEMPLATE/frontend_template.md

```diff
@@ -38,6 +38,7 @@ This section helps reviewers understand the broader context and importance of th
 <!--- If one or more lines do not apply, use ~ to ~strikethrough~ the whole line -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] My code follows the code style of this project.
+- [ ] I have added required unit tests.
 - [ ] I have run tests locally (unit tests).
```

Comment:
I think it makes sense to have only one of the two. Maybe merge them to says somethink like:
```suggestion
- [ ] I have added / updated / run required unit tests.
```

Reply:
running tests could mean verifying you didnt break someone else's 
and adding tests is our effort to test new/updated code
both make sense, and having a single checkbox covers the need well while keeping our template simple
i like it!

Reply:
**Just a thought**: It seems a bit redundant to ask if tests have been run locally since we have GitHub Actions that automatically run them on every PR. Plus, we can’t merge into `main` without a PR anyway.

Maybe just saying, “I have added/updated unit tests,” could cover it.